### ISSUE TITLE
Support for wayland unstable protocol linux_dmabuf_unstable_v1

### ISSUE THREAD

**Issue by Th3Whit3Wolf** [vaxerski/Hyprland#33]
**Describe in detail your idea.**

Protocol: [linux_dmabuf_unstable_v1](https://github.com/wayland-project/wayland-protocols/blob/main/unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml)

Whenever I run `mpvpaper -o "--loop-playlist -v" DP-3 ~/Pics/wallpaper/gifs/lazy_mario.gif` I get the following output.

```bash
Cannot load libcuda.so.1
 (+) Video --vid=1 (gif 1920x1080 10.000fps)
VO: [libmpv] 1920x1080 bgra
wl_display@1: error 1: invalid arguments for zwp_linux_buffer_params_v1@3384.add
```

######  [Ignore the libcuda error](https://github.com/GhostNaN/mpvpaper/issues/2)

[mpvpaper](https://github.com/GhostNaN/mpvpaper) will play the gif and then after the gif is over it crashes (the `loop-playlist` flag should loop the gif).

**Comment by vaxerski** [comment]
linux_dmabuf_unstable_v1 is supported by wlroots, and thus by Hyprland with `wlr_export_dmabuf_manager_v1`, no?

**Comment by vaxerski** [comment]
ok nvm done in 05d7afe2ecd69169045248e04bf039bdae2508f6 and seems to work for me too, try now

**Comment by Th3Whit3Wolf** [comment]
Well this is weird, now I get an error 

```bash
[-] Failed to swap egl buffers 0x300D
```

I'm not sure why this would occur, my computer uses an AMD Radeon RX 580X.

**Comment by Th3Whit3Wolf** [comment]
Probably worth mention I have tested this in another wlroots window manager (river) and it does work there.

**Comment by vaxerski** [comment]
ok, I'll look into this

**Comment by vaxerski** [comment]
check if fixed in 24df15105ee2fc7f2975ef2958c2e7b1258569d1, for me it worked without any issues, but I ran the test for only a minute.

**Comment by Th3Whit3Wolf** [comment]
Works now! Thank you
